modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -142,8 +143,8 @@ registry.redhat.io/openshift4/cnf-tests-rhel8:{product-version} \
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e DISCOVERY_MODE=true -e FEATURES=performance -e IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000/cnftests \
-cnf-tests-local:latest /usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+-e LATENCY_TEST_RUNTIME=<time in seconds> \
+-e IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000/cnftests cnf-tests-local:latest /usr/bin/test-run.sh  --ginkgo.v
```

Comment:
Is the double space intentional before --ginkgo.v intentional?

Reply:
corrected